wallet/core/src/derivation/mod.rs

```diff
@@ -106,7 +105,7 @@ impl AddressManager {
 
     fn create_address(&self, keys: Vec<secp256k1::PublicKey>) -> Result<Address> {
         let address_prefix = self.wallet.address_prefix()?;
-        create_address(self.minimum_signatures, keys, address_prefix, self.ecdsa, Some(self.account_kind))
+        create_address(self.minimum_signatures, keys, address_prefix, self.ecdsa, Some(self.account_kind.clone()))
```

Comment:
It no longer is.  It is now a wrapper around `String`...  and I broke my head re-implementing it as it needs to be declarable as static/const but can't be `&'static` because it needs to `impl Deserialize` so now it is `AccountKind(String)` with a whole bunch of `From` and `TryFrom` helpers.  As such, while most places I switched to use references some remain requiring explicit clones.

Reply:
Following side discussions, `AccountKind` has been refactored to use `fixedstr` and is now `Copy` again!  This is just awesome :).  Thanks for identifying this.  All relevant clones are removed since the type became copy again.  Changes pushed.